lib/managers/files.js

```diff
@@ -560,6 +560,22 @@ Files.prototype.deleteMetadata = function(fileID, scope, template, callback) {
 };
 
 /**
+ * Retrieves a file that has been moved to the trash.
+ *
+ * API Endpoint: '/files/:fileID/trash'
+ * Method: GET
+ *
+ * @param {string} fileID - The ID of the file being requested
+ * @param {Function} callback - Called with nothing if successful, error otherwise
+ * @returns {void}
+ */
+Files.prototype.getTrashedFile = function(fileID, callback) {
```

Comment:
Yes, it does:

```
12:09 $ curl https://api.box.com/2.0/files/68475254229/trash?fields=id,name -H "Authorization: Bearer ACCESS_TOKEN"
{"type":"file","id":"68475254229","etag":"2","name":"My Document.pdf"}
```

In general, almost all GET requests accept the fields parameter, even if it isn't explicitly mentioned.